recipes-multimedia/gstreamer/gstreamer1.0_1.24.0.imx.bb

```diff
@@ -92,14 +92,21 @@ LIC_FILES_CHKSUM = " \
 "
 
 # Use i.MX fork of GST for customizations
-SRC_URI:remove = "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-${PV}.tar.xz"
+SRC_URI:remove = "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-${PV}.tar.xz \
+                file://0001-tests-respect-the-idententaion-used-in-meson.patch \
+                file://0002-tests-add-support-for-install-the-tests.patch \
+                file://0003-tests-use-a-dictionaries-for-environment.patch;striplevel=3 \
+                file://0004-tests-add-helper-script-to-run-the-installed_tests.patch;striplevel=3 \
+"
```

Comment:
wonder who was this tested? look like all the test patches were inadvertently removed

Reply:
Sorry, my bad, I copied the recipe from meta-imx layer and they explicity removed these patches there. But looking at the patch and the MR opened at gstreamer upstream, I have added them back.

The only thing is that I had to rework them, because they were not applying anymore. I also removed the "striplevel=3" from patches 0003 and 0004 because this was making them fail to apply. Please let me know if something else is needed, thank you.

Reply:
I am not aware of the meta-imx but these patches are still available upstream in oe-core. They are so we can run the unit tests with the oe-core ptest infrastructure. 
All of these [patches](https://github.com/openembedded/openembedded-core/blob/662396533177b72cc1d83e95841b27f7e42dcb20/meta/recipes-multimedia/gstreamer/gstreamer1.0_1.24.5.bb#L18-L24) are still in use and mandatory upstream in oe-core otherwise we can't run the unit test on the target using the `ptest`.

Reply:
All right, thanks for the explanation. So let's keep them, not sure why meta-imx removed.